### PR TITLE
Fixed boundary selection dropdown issue, fixed privacy policy text color

### PR DESCRIPTION
**Pull request by rachana-egov**

### CODE REVIEW

File: apps/health_campaign_field_worker_app/lib/pages/login.dart [hunk 1]
```diff
@@ -226,40 +228,46 @@ class _LoginPageState extends LocalizedState<LoginPage> {
...
-    _password: FormControl<String>(
-      validators: [Validators.required],
-      value: '',
-    ),
-    _privacyCheck: FormControl<bool>(
-      value: false,
-    )
-  });
+        _userId: FormControl<String>(
+          value: 'A0352172',
```
Comment (naveenr-egov) [inline apps/health_campaign_field_worker_app/lib/pages/login.dart:232]: Remove the credentials

File: apps/health_campaign_field_worker_app/pubspec.lock [hunk 2]
```diff
@@ -501,10 +501,9 @@ packages:
   digit_data_model:
     dependency: "direct main"
     description:
-      name: digit_data_model
-      sha256: "0b81e96636496b56b4f9c5d918690a17d8985d6b131e0ab54968eb5471ee4f54"
-      url: "https://pub.dev"
-    source: hosted
+      path: "../../packages/digit_data_model"
+      relative: true
```
Comment (naveenr-egov) [inline apps/health_campaign_field_worker_app/pubspec.lock:505]: Get the packages from pub

**Review by naveenr-egov — CHANGES_REQUESTED** [review]
Resolve comments

**Comment by naveen-egov** [comment]
Closing this PR as [NEW PR](https://github.com/egovernments/health-campaign-field-worker-app/pull/598 ) created

**Pull request closed without merging**